DFC.ServiceTaxonomy.GraphSync/Notifications/GraphSyncNotifier.cs

```diff
@@ -134,6 +136,14 @@ private string GetContentTypeDisplayName(ContentItem contentItem)
 
             string exceptionText = GetExceptionText(exception);
 
+            //publish to slack
+            _slackMessagePublisher.SendMessageAsync(userMessage);
+
+            if (!string.IsNullOrWhiteSpace(exceptionText))
+            {
+                _slackMessagePublisher.SendMessageAsync($"```{exceptionText}```");
```

Comment:
Again, does this send 2 messages?
Can we send all the notification details, such as traceId and technical message? Similar to `clipboardCopy`, but without truncating the exception (and quoting it).